src/webgpu/api/validation/copyBetweenLinearDataAndTexture.ts

```diff
@@ -0,0 +1,115 @@
+import { kTextureFormatInfo } from '../../capability_info.js';
+
+import { ValidationTest } from './validation_test.js';
+
+export enum TestMethod {
+  WriteTexture = 'WriteTexture',
+  CopyBufferToTexture = 'CopyBufferToTexture',
+  CopyTextureToBuffer = 'CopyTextureToBuffer',
+}
+
+export const kAllTestMethods = [TestMethod.WriteTexture, TestMethod.CopyBufferToTexture, TestMethod.CopyTextureToBuffer];
```

Comment:
Is the enum still helpful? you could use:

```suggestion
export const kAllTestMethods = ['WriteTexture', 'CopyBufferToTexture', 'CopyTextureToBuffer'] as const;
```

(the `as const` is needed to constrain the types instead of generalizing to `string`).